internal/pkg/api/handlePGPRequest.go

```diff
@@ -52,10 +53,9 @@ func (pt *PGPRetrieverT) handlePGPKey(zlog zerolog.Logger, w http.ResponseWriter
 		return ErrTLSRequired
 	}
 	key, err := authAPIKey(r, pt.bulker, pt.cache)
-	if err != nil {
-		return err
+	if err == nil {
```

Comment:
Asking out of curiosity; what is the reason we are leaving the authAPIKey call in here?

Reply:
in order to log the api key id if one is present